crates/rust-analyzer/src/vfs_glob.rs

```diff
@@ -87,7 +91,7 @@ fn test_globs() {
 
     let filter = RustPackageFilterBuilder::default()
         .set_member(true)
-        .exclude(Glob::new("src/llvm-project/**").unwrap())
+        .exclude(std::iter::once(Glob::new("src/llvm-project/**").unwrap()))
```

Comment:
By the may, this also works:
```rust
        .exclude(Some(Glob::new("src/llvm-project/**").unwrap()))
```
But it looks very hacky :D

Reply:
`.exclude(&[Glob::new("src/llvm-project/**")])`?

Reply:
Nope, there is not into_iter for [T; N]

Reply:
There's one for `&[T]`, but I guess it's not worth it.

Reply:
`&[T]` yields only refs `&T`, but here we need `T`